CMakeLists.txt

```diff
@@ -101,6 +101,10 @@ endif()
 # Installation logic...
 #-----------------------------------------------------------------------------------------------
 
+if (NOT APPLE AND NOT WIN32)
```

Comment:
Please select which systems you want, not which you don't want.

Reply:
There is no `if(LINUX)` in CMake. Only `if(UNIX)`, but is set for both Linux and Apple.

Reply:
No wonder no one used CMAKE back in the old days.
`if (unix and not apple)` isn't any better.